pkg/utils/utils.go

```diff
@@ -14,3 +22,70 @@ func GetSocket(hostIP string, hostPort int) string {
 	}
 	return socket
 }
+
+func IskernelOlderthan514() bool {
```

Comment:
maybe a couple of unit tests for the new functions here, making sure it's bullet-proof? :-)

Reply:
done